### PR TITLE
Cleanup RedrawWindow/InvalidateRect interop

### PR DESCRIPTION
**Pull request by hughbe**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/1719)

### CODE REVIEW

File: src/Common/src/Interop/User32/Interop.RedrawWindow.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            RDW_ALLCHILDREN = 0x0080,
+            RDW_ERASENOW = 0x0200,
+            RDW_UPDATENOW = 0x0100,
+            RDW_FRAME = 0x0400,
+            RB_INSERTBANDA = (0x0400 + 1),
+            RB_INSERTBANDW = (0x0400 + 10)
+        }
+
+        [DllImport(Libraries.User32, ExactSpelling = true)]
+        public unsafe static extern BOOL RedrawWindow(IntPtr hWnd, RECT *lprcUpdate, IntPtr hrgnUpdate, RedrawWindowOptions flags);
```
[the remark this answers is not in the document]
Comment (hughbe) [reply src/Common/src/Interop/User32/Interop.RedrawWindow.cs]: Nope this is used somewhere where we have a HWND Handle without a Wrapper

File: src/Common/src/Interop/User32/Interop.RedrawWindow.cs [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class User32
+    {
+        [Flags]
+        public enum RedrawWindowOptions : uint
+        {
+            RDW_INVALIDATE = 0x0001,
```
[the remark this answers is not in the document]
Comment (hughbe) [reply src/Common/src/Interop/User32/Interop.RedrawWindow.cs]: Even though they're not used?

**Comment by hughbe** [comment]
Good to merge? I feel like my work is being duplicated by you guys so want to get this in before someone sends in the same PR. Let me know if there's a better way we can coordinate...

**Comment by zsd4yr** [comment]
> Good to merge? I feel like my work is being duplicated by you guys so want to get this in before someone sends in the same PR. Let me know if there's a better way we can coordinate...

@hughbe I think the best way forward here is to make issues for different cleanup areas in the (Un)(Safe)NativeMethods and other Interop, and then you can others who are working in this area can assign yourselves / comment on the issue to indicate you intend to work on it.

File: src/Common/src/Interop/User32/Interop.RedrawWindow.cs [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class User32
+    {
+        [Flags]
+        public enum RedrawWindowOptions : uint
```
Comment (RussKie) [inline src/Common/src/Interop/User32/Interop.RedrawWindow.cs]: Why is it not in a separate file?

**Pull request merged**